src/web/services/instructor.service.ts

```diff
@@ -122,11 +122,11 @@ export class InstructorService {
     }
 
     if (queryParams.instructorRole) {
-      paramMap.sectionname = queryParams.instructorRole;
+      paramMap.instructorrole = queryParams.instructorRole;
```

Comment:
The change is for?

The url param should all be in lower case?

Reply:
This change is a typo in the current code. 

The line above already (lines 120-122) handles the sectionname case
```
 if (queryParams.sectionName) {
      paramMap.sectionname = queryParams.sectionName;
 }
```